firebase-common/src/commonMain/kotlin/dev/gitlive/firebase/decoders.kt

```diff
@@ -28,7 +28,7 @@ fun <T> decode(strategy: DeserializationStrategy<T>, value: Any?, decodeDouble:
 
 expect fun FirebaseDecoder.structureDecoder(descriptor: SerialDescriptor): CompositeDecoder
 
-class FirebaseDecoder(internal val value: Any?) : Decoder {
+class FirebaseDecoder(val value: Any?) : Decoder {
```

Comment:
made public so custom serializers could use it. (not feeling bad about it because FirebaseEncoder.value is public too :)